collections/_events/2023-10-26-summit-nescala.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: event
+
+title: "Typelevel Summit NEScala"
+short_title: "Typelevel Summit"
+date_string: "October 26, 2023"
+location: "Online"
+description: "One day of recorded talks and one day of unconference, co-located with NE Scala."
+
+---
+
+## About the Summit
+
+The tenth Typelevel Summit will once again be co-hosted with the [Northeast Scala Symposium](http://www.nescala.org/) virtually, with one day of recorded talks and one day of (shared) unconference.
+
+The schedule for this year is as follows:
+
+* October 26 (Thursday): Typelevel Summit
+* October 27 (Friday): NE Scala 2020
```

Comment:
```suggestion
* October 27 (Friday): NE Scala 2023
```
Do you know where you copied this from?  Sounds like we might need to fix something somewhere.